src/views/answerView.js

```diff
@@ -1,34 +1,32 @@
 'use strict';
-import {SKIP_THE_QUESTION,
-  ANSWERS_LIST_ID,
-  NEXT_QUESTION_BUTTON_ID,
-   } from '../constants.js';
+
 import { quizData } from '../data.js';
-import { nextQuestion} from '../pages/questionPage.js';
+import { nextQuestion } from '../pages/questionPage.js';
 
 /**
  * Create an Answer element
  * @returns {Element}
  */
 let score = 0
 
- export const createAnswerElements = () => {
-  const currentQuestion = quizData.questions[quizData.currentQuestionIndex];
-  const correctAnswer = currentQuestion.correct;
-
-    let correctAnswerScore = document.createElement('span')
-    correctAnswerScore.textContent = 0;
-    const correctAnswerMessage = document.createElement('div')
-    correctAnswerMessage.classList.add('message')
-    correctAnswerMessage.textContent = `You have answered ${score} question correct`
-
-  const answersListElement = document.getElementById(ANSWERS_LIST_ID);
-  let answerClicked = false;
+ export const createAnswerElements = (answersListElement, unableTheButton) => {
```

Comment:
`unableTheButton` should be called `skipButton` to make it easier to understand the code.